README.md

```diff
@@ -297,60 +281,209 @@ The complete list of transformations supported and their usage in ImageKit can b
+| setRetryBackoff(interval: Long, policy: UploadPolicy.BackoffPolicy) | UploadPolicy.Builder | Sets the backoff interval in milliseconds and policy (`UploadPolicy.BackoffPolicy.LINEAR` or `UploadPolicy.BackoffPolicy.EXPONENTIAL`) for retry attempts. Defaults to interval of 1000ms and policy of `BackoffPolicy.linear`. This increases the gap between each upload retry in either linear or exponential manner. E. g. if the `interval` is set to 3 seconds, then delay for each retry (n) will increase as following: <ul><li>With linear backoff: 3, 6, 9, 12 seconds.. (`interval` * n).</li><li>With exponential backoff: 3, 6, 12, 24 seconds.. (`interval` * 2<sup>n-1</sup>).</li></ul> |
+
+Example code
+```swift
+let policy = UploadPolicy.Builder()
+    .requireNetworkType(UploadPolicy.NetworkType.UNMETERED)
+    .requiresCharging(true)
+    .setMaxRetries(5)
+    .setRetryBackoff(60000, UploadPolicy.BackoffPolicy.EXPONENTIAL)
+    .build()
+```
+
+## Upload preprocessing
+### Image preprocessing
+The `ImageUploadPreprocessor` class encapsulates a set of methods to apply certain transformations to an image before uploading. This will create a copy of the selected image, which will be transformed as per the given parameters before uploading.
+
+`ImageUploadPreprocessor.Builder` class is responsible for building the ImagePreprocess instances. This class provides following methods to access and modify the policy parameters:
+
+| Parameter | Type | Description |
+|:------------------------------------------------------------------------------------------------------------------------|:--------------------------|:-------------------------------------------------------------------------|
+| limit(width: Int, height: Int) | ImageUploadPreprocessor.Builder | Specifies the maximum width and height of the image |
+| crop(p1: Point, p2: Point) | ImageUploadPreprocessor.Builder | Specifies the two points on the diagonal of the rectangle to be cropped. |
+| format(format: ImageUploadPreprocessor.OutputFormat) | ImageUploadPreprocessor.Builder | Specify the target image format. Possible values are `ImageUploadPreprocessor.OutputFormat.JPEG` and `ImageUploadPreprocessor.OutputFormat.PNG`. Defaults to `.PNG` |
+| rotate(degrees: Float) | ImageUploadPreprocessor.Builder | Specify the rotation angle of the target image. |
+
+Example code
+```swift
+let preprocessor = ImageUploadPreprocessor.Builder()
+   .limit(width: 400, height: 300)
+   .rotate(degrees: 45)
+   .format(format: .JPEG)
+   .build()
+```
+
+### Video preprocessing
+The `VideoUploadPreprocessor` class encapsulates a set of methods to apply certain transformations to an image before uploading. This will create a copy of the selected image, which will be transformed as per the given parameters before uploading.
+
+`VideoUploadPreprocessor.Builder` class is responsible for building the ImagePreprocess instances. This class provides following methods to access and modify the policy parameters:
```

Comment:
This line is also incorrect

`VideoUploadPreprocessor.Builder class is responsible for building the ImagePreprocess instances`

Reply:
Fixed the class reference